Fhi.HelseId.Blazor/FhiHeaderDelegationHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Web;
+
+namespace Fhi.HelseId.Blazor;
+
+public class FhiHeaderDelegationHandler : DelegatingHandler
+{
+    protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+    {
+        var headers = request
+            .Headers
+            .ToDictionary(x => x.Key, x => x.Value.First());
+
+        foreach (var header in headers)
+        {
+            if (header.Key.StartsWith("fhi-"))
+            {
+                request.Headers.Remove(header.Key);
+                request.Headers.Add(header.Key, HttpUtility.HtmlEncode(header.Value));
```

Comment:
Shouldn't headers be URL-encoded instead of HTML-encoded?

Reply:
Det er ingen standard for å encode headers, så det er litt opp til oss selv.
Jeg fikk for HtmlEncode, siden den tillatter flere tegn en UrlEncode, og ikke minst siden mellomrom ikke encodes.

Testbyen Legekontor Øst
Testbyen Legekontor &#216;st (HttpUtility.HtmlEncode)
Testbyen+Legekontor+%c3%98st (HttpUtility.UrlEncode)

Reply:
Grunnen til at jeg tenkte på UrlEncode instedenfor er at HTML ikke er en del av HTTP-speccen. En positiv side med UrlEncode er at æøå blir escaped. Er litt usikker på om en LB kan finne på noe weird  dersom vi har ÆØÅ i header-verdiene (eller om det fort kan bli andre encoding issues).

Reply:
Ser at github formaterte ø'en tilbake til vanlig tegn :D ø'en blir riktig encodet med HtmlEnode, men mellomrom bevares. Tanken min er at vi kun vil formatere ikke-støttede-tegn, da det er bedre å få encodet data enn at requesten feiler, men det er dumt å få "rar" encodet data om ingen ting skulle feilet.